tests/e2e/utils.go

```diff
@@ -90,3 +97,61 @@ func getActiveApplication() string {
 	odoActiveApp := strings.Split(reActiveApp.FindString(stdOut), "*")[1]
 	return strings.TrimSpace(odoActiveApp)
 }
+
+func retryingForOutputMatchStringOfHTTPResponse(url, match string, retry, sleep int) bool {
+	for i := 0; i < retry; i++ {
```

Comment:
same here, use simple for loop instead of initializing variable

Reply:
```retry``` is not an array of string. so can not be operated through a range.